libs/server/Storage/Functions/MainStore/RMWMethods.cs

```diff
@@ -413,6 +415,28 @@ private bool InPlaceUpdaterWorker(ref SpanByte key, ref SpanByte input, ref Span
                     rmwInfo.Action = RMWAction.ExpireAndStop;
                     return false;
 
+                case RespCommand.GETEX:
+                    CopyRespTo(ref value, ref output);
+
+                    if (input.ExtraMetadata > 0)
+                    {
+                        byte* pbOutput = stackalloc byte[ObjectOutputHeader.Size];
```

Comment:
What's this for?

Reply:
It's to call existing EvaluateExpireInPlace method but I don't care about the output param, so created a temp SpanByteAndMemory without any allocation and called the existing method

Reply:
But you're not calling EvaluateExpireInPlace with this variable...

Reply:
I am using the below code
```
var _ = new SpanByteAndMemory(SpanByte.FromPinnedPointer(**pbOutput**, ObjectOutputHeader.Size));
```

Reply:
Mmm ok I wasn't familiar with the syntax...

Reply:
Why not something like:
```cs
byte* pbOutput = stackalloc byte[ObjectOutputHeader.Size];
var _output = new SpanByteAndMemory(pbOutput, ObjectOutputHeader.Size);
EvaluateExpireCopyUpdate(ExpireOption.None, expiryExists: oldValue.MetadataSize > 0, ref input, ref oldValue, ref newValue, ref _output);
```

It'll make what you're doing here clearer and more in-line with our code.

Reply:
Fixed